director/jobs/templates/jobs_list_job.html

```diff
@@ -0,0 +1,48 @@
+<div class="job has-text-black">
+  <div class="job-column job-summary">
+    <span class="job-id is-block leading-1">{% if job.id %}JOB {{job.id}}{% endif %}</span>
+    <span class="tag is-uppercase job-tag {{job.colour}}">{{job.status}}</span>
+    <div class="job-icon status is-flex">
+      <i data-feather="{{job.icon}}" class="icon flex-shrink-0"></i>
+      <span class="text truncate has-text-weight-bold">{{job.method_label}}</span>
+    </div>
+  </div>
+
+  <div class="job-column job-details flex-grow is-flex flex-col">
+    <header class="job-details-header flex-wrap is-flex mb-2">
+
+      <span class="job-info job-icon job-user is-flex">
```

Comment:
For each of these `.job-info` elements, as a user I'd like to have a `title` attribute so that I could hover over them and see which property of the job they were referring to. e.g in this case `title="User who created the job"`

Reply:
Will do.